src/scripts/chimera-dome

```diff
@@ -46,6 +48,18 @@ class ChimeraDome (ChimeraCLI):
                                 help="Tell the dome to follow TELESCOPE when tracking"
                                 "(only utilized when using --track"))
 
+        self.addHelpGroup("CALIBRATIONLAMP", "CalibrationLamp")
+        self.addParameters(dict(name="calibrationlamp",
+                                default="/CalibrationLamp/0",
+                                helpGroup="CALIBRATIONLAMP",
+                                help="Calibration lamp to be used."))
+
+        self.addParameters(dict(name="intensity",
+                                default=None,
+                                type=float,
+                                helpGroup="CALIBRATIONLAMP",
+                                help="Select calibration lamp intensity (from 0 to 100.)."))
```

Comment:
Intensity can be something different of this interval in lamps other than the fake. I think that this should come from the configuration like on the focuser.
https://github.com/astroufsc/chimera/blob/7d38c7e098c00de9c96ba9ba450f27de1985cceb/src/chimera/instruments/fakefocuser.py#L77-L78